plugins/dependencies/src/main/kotlin/de/fayard/refreshVersions/internal/OutputFile.kt

```diff
@@ -0,0 +1,44 @@
+package de.fayard.refreshVersions.internal
+
+import org.gradle.api.Project
+
+internal enum class OutputFile(var path: String, var existed: Boolean = false, val alternativePath: String? = null) {
+    OUTPUTDIR("buildSrc/src/main/kotlin"),
+    BUILD("buildSrc/build.gradle.kts", alternativePath = "buildSrc/build.gradle"),
+    GIT_IGNORE("buildSrc/.gitignore"),
+    LIBS("buildSrc/src/main/kotlin/Libs.kt"),
+    VERSIONS_KT("buildSrc/src/main/kotlin/Versions.kt"),
+    GRADLE_PROPERTIES("gradle.properties"),
+    VERSIONS_PROPERTIES("versions.properties")
+    ;
+
+    fun fileExists(project: Project) = when {
+        project.file(path).exists() -> true
+        alternativePath != null -> project.file(alternativePath).exists()
+        else -> false
+    }
+
+    fun logFileWasModified(delete: Boolean = false) {
+        logFileWasModified(path, existed, delete)
+    }
+
+    companion object {
+
+        fun logFileWasModified(path: String, existed: Boolean, delete: Boolean = false) {
+            val ANSI_RESET = "\u001B[0m"
+            val ANSI_GREEN = "\u001B[32m"
+            val ANSI_RED = "\u001B[31m"
```

Comment:
Please move these as private constants in that same companion object, below, so the IDE doesn't warn about the case not following the conventions.

Reply:
done